pkg/locspec/locations_unix_test.go

```diff
@@ -0,0 +1,22 @@
+package locspec
```

Comment:
You can't use build guards like this and the 'unix' build tag doesn't exist on most versions of go. You should check runtime.GOOS and skip as appropriate.

Reply:
Sounds good. Changing the test accordingly. Thanks for the suggestion.